ledger/src/blockstore_db.rs

```diff
@@ -1468,6 +1468,31 @@ pub struct WriteBatch {
     write_batch: RWriteBatch,
 }
 
+pub struct BlockstoreByteReference<'a> {
```

Comment:
probably does not need to be `pub`.

Reply:
done in https://github.com/anza-xyz/agave/pull/4674/commits/70bcaf74f009d5617aa55a794d55babc16fd68f1; we can make it `pub` later if we need it